### DISCUSSION TITLE
Is possible to join two tables if it don't have relationship?

### DISCUSSION THREAD

**Discussion by sheldygg** [aminalaee/sqladmin#832] [Q&A]
I want create table where I want show columns from different tables and they don't have relationship. This is possible?

**Comment by sheldygg** [comment]
In order to implement this I had to override the methods: `list_query`, `get_list_value`, `list`, `count_query`: in the `ModelView` class, and remake the template for the list, this turned out to be quite difficult :((((

**Reply by mmzeynalli** [reply]
As it should have been. Library inspect model and finds relationships, loading them to the memory. If there is no relation, it is impossible to detect that.

We could have extra param for this, but this is not optimal DB behavior, so this is not in plan
